docs/headerparser_usage.md

```diff
@@ -0,0 +1,67 @@
+## Headerparser
+
+headerparser is a tool that assists in writing device system call descriptions for Syzkaller.
+
+In order to make Syzkaller smarter when it comes to fuzzing a device node, you can provide it with
```

Comment:
s/Syzkaller/syzkaller/
we always spell it this way

Reply:
thanks Ill fix this